docs/nlnet-deliverables/4a.md

```diff
@@ -0,0 +1,38 @@
+
+# 4a. Identify risks in usability, accessibility, desirability, viability (for adoption)
+
+
+## Amazon style [Future PR](./future-PR-FAQ.md). 
+
+
+A de-risking  exercise designed to reduce waste in software delivery and ensuring your solution will address the right problem, and that you understand and can articulate the problem. It is a written narrative articulating a product/feature in a way a customer would understand and make an informed decision on whether they want to try it out.
+
+## [Story Map as of 30/06/2025](./story-map-30-06-25.jpg). 
+
+
+[Story Maps were popularised by Jeff Patton](https://jpattonassociates.com/story-mapping/) and are a de-risking exercise that helps ensure you aren't building the right thing, the wrong way. A story map is structured as:
+
+-  User activities across the top (light blue, horizontal axis)
```

Comment:
Additional space

Reply:
Fixed in https://github.com/sciety/sciety/pull/3970/commits/e46303fbb126cd34a318de521af40ce994d4ef25